src/js/providers/tracks-mixin.js

```diff
@@ -667,62 +688,127 @@ function _clearSideloadedTextTracks() {
 }
 
 function _cueChangeHandler(e) {
```

Comment:
i'm not 100% on what is happening here? is the additional logic exposing id3 information that previously we left up to hls.js?

Reply:
No. This is what triggers `"metadataCueParsed"` events when new cues are found in metadata track.cues, and `"meta"` events when new active cues in metadata track.activeCues. Previously we ignored `track.cues` and only fired meta events for active cues. A cue is active when the currentTime enters its range.

Reply:
aha! makes sense 👍